### PR TITLE
feat: ✨ add defined aspect ratios and additional colors

### LINKED ISSUE #355: docs: 📚 show tokens in Storybook
## Description

Currently the tokens aren't shown in Storybook. I experience that this is a problem for people starting with the project  + for reviewing tokens, which aren't in use.

I suggest to write automatic stories, which show...

- [ ] shadows
- [ ] z-indexs
- [ ] spacings 
- [ ] opacity
- [ ] font-size
- [ ] font-weight 
- [ ] border-radius
- [ ] risk

and also for **colors**

- [ ] error
- [ ] success

We maybe can use the following for that:
- https://storybook.js.org/docs/web-components/api/doc-block-colorpalette
- https://storybook.js.org/docs/web-components/api/doc-block-icongallery
- https://storybook.js.org/docs/web-components/writing-docs/doc-blocks#typeset

We should write some minimal docs how to e. g. use them with TailwindCSS.

The Stories should get their data automatically from the imported Tailwind Theme from the tokens package.

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by saemik94**
## Description:
closes #355 #728 

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] Documentation is created/updated
- [x] Migration Guide is created/updated
- [x] E2E tests (features, a11y, bug fixes) are created/updated
- [x] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/docs/General/07-Tokens.mdx [hunk 1]
```diff
@@ -16,9 +16,173 @@ They use shorthand RGB values (e. g. `108 194 84`) to allow setting opacity via
...
+
+## Font Sizes
+
+<table>
+  <tbody>
+    {Object.entries(theme.fontSize).map((token, i) => {
+      return (
+        <tr>
+          <td>
+            <div class="font-bold whitespace-nowrap ">sd-font-size-{token[0]}</div>
```
Comment (yoezlem) [inline packages/components/src/docs/General/07-Tokens.mdx:40]: Please remove the whitespace

File: packages/components/src/docs/General/07-Tokens.mdx [hunk 2]
```diff
@@ -16,9 +16,173 @@ They use shorthand RGB values (e. g. `108 194 84`) to allow setting opacity via
...
+
+## Font Weight
+
+<table>
+  <tbody>
+    {Object.entries(theme.fontWeight).map((token, i) => {
+      return (
+        <tr>
+          <td>
+            <div class="font-bold whitespace-nowrap ">sd-font-size-{token[0]}</div>
```
Comment (yoezlem) [inline packages/components/src/docs/General/07-Tokens.mdx:60]: https://github.com/solid-design-system/solid/pull/757/files#r1490703574

File: packages/components/src/docs/General/07-Tokens.mdx [hunk 3]
```diff
@@ -16,9 +16,173 @@ They use shorthand RGB values (e. g. `108 194 84`) to allow setting opacity via
...
+> "By default, these values are inherited by the padding, margin, width, height, maxHeight, flex-basis, gap, inset, space, translate, scrollMargin, scrollPadding, and textIndent core plugins." [tailwindcss docs](https://tailwindcss.com/docs/theme#spacing)
+
+<table>
+  <tbody>
+    {Object.entries(theme.spacing).map((token, i) => {
+      return (
+        <tr>
+          <td>
+            <div class="flex flex-col">
+              <div class="font-bold whitespace-nowrap ">sd-spacing-{token[0]}</div>
```
Comment (yoezlem) [inline packages/components/src/docs/General/07-Tokens.mdx:83]: https://github.com/solid-design-system/solid/pull/757/files#r1490703574

File: packages/components/src/docs/General/07-Tokens.mdx [hunk 4]
```diff
@@ -16,9 +16,173 @@ They use shorthand RGB values (e. g. `108 194 84`) to allow setting opacity via
...
+## Opacities
+
+<table>
+  <tbody>
+    {Object.entries(theme.opacity).map((token, i) => {
+      return (
+        <tr>
+          <td>
+            <div class="flex flex-col">
+              <div class="font-bold whitespace-nowrap ">sd-opacity-{token[0]}</div>
```
Comment (yoezlem) [inline packages/components/src/docs/General/07-Tokens.mdx:144]: https://github.com/solid-design-system/solid/pull/757/files#r1490703574

File: packages/components/src/docs/General/07-Tokens.mdx [hunk 5]
```diff
@@ -16,9 +16,173 @@ They use shorthand RGB values (e. g. `108 194 84`) to allow setting opacity via
...
+## Shadow
+
+<table>
+  <tbody>
+    {Object.entries(theme.boxShadow).map((token, i) => {
+      return (
+        <tr>
+          <td>
+            <div class="flex flex-col">
+              <div class="font-bold whitespace-nowrap ">--sd-shadow-{token[0]}</div>
```
Comment (yoezlem) [inline packages/components/src/docs/General/07-Tokens.mdx:177]: https://github.com/solid-design-system/solid/pull/757/files#r1490703574

**Comment by saemik94** [comment]
This PR also includes #728

File: packages/tokens/src/create-theme.cjs [hunk 6]
```diff
@@ -199,7 +199,36 @@ const getColors = (scope, cssVariableScope) => {
...
+    sanitizeValue('risk-increased')
+  )}, ${getColor('#FFF000')}) / <alpha-value>) /* Exclusively for marking fonds */`;
+
+  result[sanitizeValue('risk-high')] = `rgb(var(--sd-color-${sanitizeKey(sanitizeValue('risk-high'))}, ${getColor(
+    '#FA9B1E'
+  )}) / <alpha-value>) /* Exclusively for marking fonds */`;
+
+  result[sanitizeValue('risk-veryhigh')] = `rgb(var(--sd-color-${sanitizeKey(
+    sanitizeValue('risk-veryhigh')
+  )}, ${getColor('#FF0000')}) / <alpha-value>) /* Exclusively for marking fonds */`;
```
Comment (mariohamann) [inline packages/tokens/src/create-theme.cjs:228]:
```suggestion
  const colorMappings = {
  'success': {
    color: '#25880E',
    comment: 'Used for success messages'
  },
  'warning': {
    color: '#BB8D20',
    comment: 'Used for notifications'
  },
  'risk-low': {
    color: '#017DC3',
    comment: 'Exclusively for marking fonds'
  },
  'risk-moderate': {
    color: '#00A593',
    comment: 'Exclusively for marking fonds'
  },
  'risk-increased': {
    color: '#FFF000',
    comment: 'Exclusively for marking fonds'
  },
  'risk-high': {
    color: '#FA9B1E',
    comment: 'Exclusively for marking fonds'
  },
  'risk-veryhigh': {
    color: '#FF0000',
    comment: 'Exclusively for marking fonds'
  },
};

// Initialize the result object
const result = {};

// Iterate over the colorMappings object
for (const [key, { color, comment }] of Object.entries(colorMappings)) {
  // Construct the result using the key, color, and comment
  result[sanitizeValue(key)] = `rgb(var(--sd-color-${sanitizeKey(sanitizeValue(key))}, ${getColor(color)}) / <alpha-value>) /* ${comment} */`;
}
```

Could you try something like this?

**Review by mariohamann — CHANGES_REQUESTED** [review]
https://github.com/solid-design-system/solid/pull/757/files#r1495779989

This should be definitely changed.

**Comment by Vahid1919** [comment]
@saemik94 This seems to have made significant changed to some of the components (sd-teaser-media. sd-video and sd-flag) Is this intended? 

Example: 
image


You can take a look at the chromatic UI tests to look at the specifics

**Comment by saemik94** [comment]
@Vahid1919 thanks for the hint! I was able to fix it :)

**Review by Vahid1919 — APPROVED** [review]

**Review by karlbaumhauer — APPROVED** [review]

**Pull request merged**